src/client/components/Graphs.tsx

```diff
@@ -4,6 +4,16 @@ interface GraphsState {
   data: any[]
 }
 
+const parseDate = (timestamp: string): string => {
+  const dateObj = new Date(timestamp)
```

Comment:
JavaScript's `Date` object does not play nicely with daylight savings. Look into using `date-fns` instead: https://github.com/date-fns/date-fns